script/setup/addNewProject.ts

```diff
@@ -263,6 +269,50 @@ export const getProjectTokenListInfo = async (
         choices: tokenChoices,
       },
     ]);
+    const tokens = response.tokens as Tokens[];
+    const isEthAndWethPresent =
+      tokens.includes(Tokens.ETH) && tokens.includes(Tokens.WETH);
+    const isUsdcAndUsdcePresent =
+      tokens.includes(Tokens.USDC) && tokens.includes(Tokens.USDCE);
+
+    let mergeEthAndWeth = false;
+    let mergeUsdcAndUsdce = false;
```

Comment:
mergeEthAndWeth and mergeUsdcAndUsdce are being used locally immediately after assign.
so they can be consts while using instead of declaring as let here